### PR TITLE
(FB: 155149) - Fix small typo

### PR DESCRIPTION
**Pull request by benrudolph**
Mobile Users to Mobile Workers

http://manage.dimagi.com/default.asp?155149

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/users/templates/users/mobile/users_list.html [hunk 1]
```diff
@@ -170,7 +170,9 @@
         </div>
         <p class="help-block">
             <i class="icon icon-info-sign"></i>
-            Search any text, or use a targeted query. For more info see the <a href='https://wiki.commcarehq.org/display/commcarepublic/Create+and+Manage+CommCare+Mobile+Workers#CreateandManageCommCareMobileWorkers-C.SearchforMobileWorkers' target='_blank'>Mobile Users</a> help page
+            Search any text, or use a targeted query. For more info see the <a
+                href='https://wiki.commcarehq.org/display/commcarepublic/Create+and+Manage+CommCare+Mobile+Workers#CreateandManageCommCareMobileWorkers-C.SearchforMobileWorkers'
+                target='_blank'>Mobile Workers</a> help page
```
Comment (czue) [inline corehq/apps/users/templates/users/mobile/users_list.html]: this whole thing should probably be in a `blocktrans`

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Pull request merged**